examples/demo/src/shared/Title/Title.js

```diff
@@ -3,13 +3,13 @@ import { Link } from 'react-router-dom';
 import { PropTypes } from 'prop-types';
 import './Title.scss';
 
-const TitleBar = ({ backHome, children, className }) => (
+const TitleBar = ({ onClick, children, className }) => (
   <div className={className}>
     <div className="container-fluid container af-title-bar__wrapper">
-      {backHome && (
-        <Link title="Retour à l'accueil" className="btn af-btn--circle" to="/">
```

Comment:
why you don't use Link router component ?

Reply:
It was not working inside my stories ^^
I generaly Never use it.

Reply:
you just have to wrap your story by a Router
i think you also create global decorator